docs/install.rst

```diff
@@ -157,6 +157,16 @@ You then must set an Optimizely project code in ``.env``::
     You are not required to set up Optimizely as detailed above. If not configured,
     bedrock will treat the switches as set to ``off``.
 
+To do the equivalent in one of the bedrock apps deployed with `Deis <http://deis.io/>`_, use `deis config <http://docs.deis.io/en/latest/using_deis/config-application/`_. To continue the example above with a Deis app named ``bedrock-demo-switch``::
+
+    deis config:set SWITCH_FIREFOX_NEW_OPTIMIZELY=True -a bedrock-demo-switch
```

Comment:
I was just following the example immediately above-- I like `on` better though, so I'll go ahead and update both.